.scripts/build_docs.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+# abort on errors
+set -e
+
+OUTPUT_DIR=docs/docs_page
+TMP=docs/tmp
+
+#clean and copy images
+if [ -d "$OUTPUT_DIR" ]; then rm -Rf $OUTPUT_DIR; fi
+if [ -d "$TMP" ]; then rm -Rf $TMP; fi
+if [ -d "$TMP_README_DIR" ]; then rm -Rf $TMP_README_DIR; fi
+mkdir -p $OUTPUT_DIR/docs/img/
+mkdir -p $TMP/docs/
+mkdir -p $TMP/readme/
+cp docs/img/* $OUTPUT_DIR/docs/img/
+
+#Split README into sections
+.scripts/doc-preprocessor.rb README.md "$TMP/readme/README.md" --section "Velocidi iOS SDK in Objective-C" --title "Velocidi iOS SDK in Objective-C"
+.scripts/doc-preprocessor.rb README.md "$TMP/docs/Installation.md" --section "Installation" --title "Installation"
+.scripts/doc-preprocessor.rb README.md "$TMP/docs/Send a track event.md" --section "Send a track event" --title "Send a track event"
```

Comment:
Is it possible to have the filenames, instead of a space character between words, have another separator (`_` or `-`)?

Reply:
No. It will mess with the names that appear in the nav bar
![image](https://user-images.githubusercontent.com/7612420/61728957-2200db00-ad6e-11e9-94e1-c1bee5436b01.png)